### PR TITLE
[docs][python] Reinforce knowledge of `is_primary_data` in all notebooks 

### PR DESCRIPTION
**Pull request by pablo-gar**
Addresses #467 for Python. Once approved, R work follows.

### CODE REVIEW

File: api/python/notebooks/analysis_demo/comp_bio_census_info.ipynb [hunk 1]
```diff
@@ -18,6 +17,9 @@
     "- Census summary content tables\n",
     "- Understanding Census contents beyond the summary tables\n",
     "\n",
+    "⚠️ Note that the Census RNA data includes duplicate cells present across multiple datasets, these can be filtered in or out using the cell metadata variable `is_primary_data`.\n",
```
Comment (bkmartinjr) [inline api/python/notebooks/analysis_demo/comp_bio_census_info.ipynb:20]: run-on sentence.  Suggest breaking into two, e.g.,

> Note that the Census RNA data includes duplicate cells present across multiple datasets. Duplicate cells can be filtered in or out using the cell metadata variable `is_primary_data`.

I would also add a link to the normative section of the CxG schema that defines this attribute.

**Review by bkmartinjr — CHANGES_REQUESTED** [review]
suggest copyedit & link to CxG schema

**Comment by atolopko-czi** [comment]
Do we need to consider an API-level change to better support the (seemingly) common use of wanting to filter primary data?

File: api/python/notebooks/analysis_demo/comp_bio_census_info.ipynb (hunk 1 above)
Comment (atolopko-czi) [reply api/python/notebooks/analysis_demo/comp_bio_census_info.ipynb:20]: +1

**Review by atolopko-czi — CHANGES_REQUESTED** [review]

**Review by atolopko-czi — APPROVED** [review]

**Review by bkmartinjr — APPROVED** [review]

**Pull request merged**